### ISSUE TITLE
[Feature] Display list of the first 5 actors listed on TMDB for each Movie and TV Show

### ISSUE THREAD

**Issue by BeardedTek** [lardbit/nefarious#156]
One thing I would love to see is the display of the top actors in each movie.

I have no problem delving into the code to figure out what needs to be done, but would just need a bit of help locating that section.

This has become one of my favorite projects on the internet and I honestly don't know what I'd do without it. THANK YOU for your time!

**Comment by lardbit** [comment]
Yeah I think that would be a nice addition.  I've also been thinking about adding the streaming platform (e.g Netflix) so users can decide whether to download if they already have access to that service.

I'll keep this open until I get some time to work on it, but I'll include some code references if you decide to delve into it.

See [DEVELOPMENT.md](https://github.com/lardbit/nefarious/blob/master/docs/DEVELOPMENT.md) for instructions on how to build the front & back-ends.

# Getting the cast data

You'll use [The Movie Database API service](https://developers.themoviedb.org/3/getting-started/introduction) to query a movie's cast.  Here's an [example query for Ghostbusters: Afterlife](https://api.themoviedb.org/3/movie/425909/credits?api_key=21c8985a267ac3f11ea75baf2c05c3ba&language=en-US) (using nefarious' api key in the url)

nefarious uses this [python tmdb client](https://github.com/celiao/tmdbsimple) for interacting with tmdb.  

Here's a simple [python example you can model after which gets the "discover" content](https://github.com/lardbit/nefarious/blob/master/src/nefarious/api/views.py#L377-L398).

Add a new API url via https://github.com/lardbit/nefarious/blob/master/src/nefarious/api/urls.py

# Displaying the cast data

Media Results components
- https://github.com/lardbit/nefarious/blob/master/src/frontend/src/app/media-results/media-results.component.ts
- https://github.com/lardbit/nefarious/blob/master/src/frontend/src/app/media-results/media-results.component.html

Movie result component:
- https://github.com/lardbit/nefarious/blob/master/src/frontend/src/app/media/media-movie.component.ts
- https://github.com/lardbit/nefarious/blob/master/src/frontend/src/app/media/media-movie.component.html

Client API:
https://github.com/lardbit/nefarious/blob/master/src/frontend/src/app/api.service.ts

**Comment by BeardedTek** [comment]
I'll be trying to start tackling this in the next month or so once my main pet project releases its first stable version.  Thanks for all the amazing work!!!
